unstoppable-ios-app/domains-manager-ios/Entities/Wallets/UDWallet.swift

```diff
@@ -50,6 +50,17 @@ enum BlockchainType: String, CaseIterable, Codable, Hashable {
             return "Polygon"
         }
     }
+    
+    func supportedChainId(isTestNet: Bool) -> Int? {
```

Comment:
This method would belong better to `UnsConfigManager.swift`

Reply:
It's appear this function wasn't used at all. Removed